Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -280,6 +285,16 @@ struct CombinatorialKalmanFilterResult {
   PathLimitReached pathLimitReached;
 };
 
+/// Prepare the track container for the CombinatorialKalmanFilter
+///
+/// @param trackContainer The track container to be prepared
+template <typename track_container_t>
+static void prepareTrackContainerForCombinatorialKalmanFilter(
+    track_container_t& trackContainer) {
+  trackContainer.template addColumn<CombinatorialKalmanFilterTipState>(
+      s_combinatorialKalmanFilterTipStateColumn);
```

Comment:
As @paulgessinger noticed: This does not work with all backend because some only support a subset of primitive data types but definitely not custom PODs. We have two options here: Either fan out the properties or go back to store these info in a vector